README.md

```diff
@@ -423,19 +409,30 @@ pa11y('http://example.com/', {
 
 ### `ignore` (array)
 
-An array of result codes and types that you'd like to ignore. You can find the codes for each rule in the console output and the types are `error`, `warning`, and `notice`.
+An array of result codes and types that you'd like to ignore. You can find the codes for each rule in the console output and the types are `error`, `warning`, and `notice`. Note: `warning` and `notice` messages are ignored by default.
 
 ```js
 pa11y('http://example.com/', {
     ignore: [
-        'notice',
         'WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2'
     ]
 });
 ```
 
 Defaults to an empty array.
 
+### `includeNonErrors` (boolean)
+
+Whether to include non-error results in the Pa11y report. Issues with a type of `warning` or `notice` are not directly actionable and so they are excluded by default. You can include them by using this option:
+
```

Comment:
what do i do if i want warnings but not notices?

Reply:
You'd do: ```js
pa11y('http://example.com/', {
    includeNonErrors: true,
    ignore: ['notice']
});
```

Reckon we need to be documented explicitly, or should we wait and see if anyone asks how to do it?

Reply:
always document explicitly 😁

Reply:
done 👍